spec/tree_spec.rb

```diff
@@ -1,14 +1,77 @@
 require 'rspec'
 require 'tree'
 
-describe 'Tree' do
+describe 'AppleTree' do
```

Comment:
You can remove the quotes here. If you do that, you can reference AppleTree anywhere with ``described_class``